packages/fxa-shared/metrics/glean/web/index.ts

```diff
@@ -112,10 +112,9 @@ export const eventsMap = {
   },
 
   thirdPartyAuth: {
-    viewWithNoPasswordSet: 'third_party_auth_login_no_pw_view',
-
     startGoogleAuthFromReg: 'google_oauth_reg_start',
 
+    viewWithNoPasswordSet: 'third_party_auth_login_no_pw_view',
```

Comment:
Should these be sorted with login for consistency, or doesn't matter?

Reply:
It's listed in the Google sheet for third party auth so I'm going to leave it.